src/Chainweb/Pact/Backend/PactState.hs

```diff
@@ -104,8 +108,8 @@ getLatestBlockHeight db = do
 getPactTableNames :: Database -> IO (Vector Utf8)
 getPactTableNames db = do
   let sortedTableNames :: [[SType]] -> [Utf8]
-      sortedTableNames rows = M.elems $ M.fromListWith const $ flip List.map rows $ \case
-        [SText u] -> (Text.toLower (fromUtf8 u), u)
+      sortedTableNames rows = List.sortOn fromUtf8 $ flip List.map rows $ \case
```

Comment:
this is just a way more intuitive way to sort these table names. IDK why I never changed this code before. Lars was really confused when staring at it.